BITCORNService/Models/CornTx.cs

```diff
@@ -14,8 +14,8 @@ public CornTx()
         public string Platform { get; set; }
         public string TxType { get; set; }
         public decimal? Amount { get; set; }
-        public string SenderId { get; set; }
-        public string ReceiverId { get; set; }
+        public int? SenderId { get; set; }
+        public int ReceiverId { get; set; }

```

Comment:
Why are we changing this?

Reply:
corntx should always point to user primary key